kimchi/src/plonk_sponge.rs

```diff
@@ -129,6 +134,22 @@ impl<Fr: PrimeField> FrSponge<Fr> for DefaultFrSponge<Fr, SC> {
         if let Some(runtime_lookup_table) = runtime_lookup_table.as_ref() {
             points.push(runtime_lookup_table)
         }
+        if let Some(runtime_lookup_table_selector) = runtime_lookup_table_selector.as_ref() {
```

Comment:
Given that we have now a lot of these, may something like this will be more readable:
```rust
        let selectors = [
            range_check0_selector,
            range_check1_selector,
            foreign_field_add_selector,
        ];
        for selector in selectors.into_iter().filter_map(Option::as_ref) {
            points.push(selector);
        }
```

Reply:
I would tend to not create intermediate structures but that's me ;-)